setup.md

```diff
@@ -0,0 +1,134 @@
+# Project Setup Guide
+
+Welcome to the LaminarFlow! This guide walks you through the full setup required to run the application locally or in a development environment.
+
+---
+## Prerequisites
+Ensure you have the following installed:
+
+*  **Node.js** ≥ v18+
+
+*  **pnpm** ≥ v8+
+
+Install it with:
+```bash
+
+npm install -g pnpm
+```
+
+* PostgreSQL or compatible DB (used by Prisma)
+---
+## Directory Structure Overview
+
+```text
+
+/app             - App routes, API, UI components
+/components      - Reusable UI elements
+/lib             - Utilities (CSV export, invoice agents)
+/trpc            - Backend procedures and logic
+/styles          - Tailwind and global styles
+```
+---
+
+## Getting started
+### 1. Clone the repository
+```bash
+
+git clone <your-repo-url>
+cd your-project
+```
+### 2. Install dependencies
+```bash
+
+pnpm install
+```
+### 3. Set up your environment variables
+Create a `.env` file at the root of your project. You can copy from `.env.example` if provided, and define the following:
+
+```dotenv
+
+DATABASE_URL=postgresql://user:password@localhost:5432/dbname
+LANGBASE_API_KEY=your_langbase_key
+GOOGLE_API_KEY=your_google_api_key
+ARCJET_KEY=your_arcjet_key
+```
+### 4. Run the database migration
+```bash
+
+pnpx prisma migrate dev --name init
+```
+You can also introspect the existing DB schema if needed:
+```bash
+
+pnpx prisma db pull
+```
+### 5. Start the development server
+```bash
+
+pnpm dev
+```
+
+Visit [Preview Website](http://localhost:3000) to view the app.
+
+---
+

```

Comment:
do update the features section, for more details, use the product to understand how LaminarFlow works